devicon.json

```diff
@@ -6103,13 +6779,16 @@
         "aliases": []
     },
     {
-        "name": "vuetify", 
+        "name": "vuetify",
+        "altnames": [
+            "visualstudiocode"
```

Comment:
`vuetify` has nothing in common with `visualstudiocode`

Reply:
3c5bf839850823623296b0f0a222ba65aff6386c